baseframe/forms/validators.py

```diff
@@ -402,11 +413,11 @@ class ValidUrl(object):
     )
 
     def __init__(self, message=None, message_urltext=None, invalid_urls=[]):
-        self.message = message or _(
+        self.message = message or __(
             u'The URL “{url}” is not valid or is currently inaccessible'
         )
         self.invalid_urls = invalid_urls
-        self.message_urltext = message_urltext or _(
+        self.message_urltext = message_urltext or __(
             u'The URL “{url}” linked from “{text}” is not valid or is currently inaccessible'
         )
```

Comment:
In this case `__` is the correct choice as `__init__` is called at load time.

Reply:
Except these too should be on the class as `default_message` and `default_message_urltext`.